scripts/resultsSum.pl

```diff
@@ -274,6 +275,10 @@ sub resultReporter {
 		$AQACertContent =~ s/\n/\n# /g;
 		print $fhOut "# AQACert.log content: \n# " . $AQACertContent . "\n";
 
+		#add machine name in TAP file
+		$host = hostname;
```

Comment:
Should be `my $host = hostname;`
https://ci.adoptium.net/view/Test_grinder/job/Grinder/7051/console